src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -124,12 +167,58 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    stopDefaultEvent = (e: React.KeyboardEvent): void => {
+        e.preventDefault();
+        e.stopPropagation();
+    };
+
+    setPopupListActiveItem = (index: number): void => this.setState({ activeItemIndex: index });
+
+    handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>): void => {
+        const { activeItemIndex, popupItems, popupReference } = this.state;
+
+        if (!popupReference) {
+            return;
+        }
+
+        const itemsCount = popupItems.length;
+
+        switch (event.key) {
+            case 'ArrowDown':
+                if (!itemsCount) {
+                    break;
+                }
+                this.stopDefaultEvent(event);
+                this.setPopupListActiveItem(activeItemIndex === itemsCount - 1 ? 0 : activeItemIndex + 1);
+                break;
+            case 'ArrowUp':
+                if (!itemsCount) {
+                    break;
+                }
+                this.stopDefaultEvent(event);
+                this.setPopupListActiveItem(activeItemIndex === 0 ? itemsCount - 1 : activeItemIndex - 1);
+                break;
+            case 'Enter':
+                this.stopDefaultEvent(event);
+                this.handleSelect(activeItemIndex);
+                this.setPopupListActiveItem(0);
+                break;
+            default:
+            // don't stop default event if not above keys
+        }
+    };
+
     render(): JSX.Element {
-        const { className, isDisabled, placeholder, ...rest } = this.props;
-        const { editorState, popupReference } = this.state;
+        const { className, isDisabled, itemRow, placeholder, ...rest } = this.props;
+        const { activeItemIndex, editorState, popupItems, popupReference } = this.state;
 
         return (
-            <div className={classnames(className, 'ba-ReplyField')}>
+            <div
+                className={classnames(className, 'ba-ReplyField')}
+                onKeyDown={this.handleKeyDown}
+                role="textbox"
```

Comment:
This may be a little confusing, since it's not what the user is actually interacting with. Can the `onKeyDown` handler be applied to the Editor instead?